### PR TITLE
Add a few comments to justfile to make it easier for new developers

### PR DESCRIPTION
**Pull request by chamons**
While working on https://github.com/casey/just/pull/1118 I ran into a few issues on my Windows machine. I added a few comments to the justfile that would have helped past me.

If you don't like them, feel free to reject the PR, no feelings will be hurt.

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -2,6 +2,9 @@
 # ^ A shebang isn't required, but allows a justfile to be executed
 #   like a script, with `./justfile test`, for example.
 
+# Windows users will need to enable this unless running under something like Git for Windows, GitHub Desktop, or Cygwin.
+#set windows-powershell := true
```
Comment (casey) [inline justfile]: Let's just make it the default, I think it's probably better that way:

```suggestion
set windows-powershell := true
```

**Comment by casey** [comment]
Nice, just one comment: I think we should just set windows-powershell instead of adding a comment.

File: justfile [hunk 2]
```diff
@@ -2,6 +2,8 @@
 # ^ A shebang isn't required, but allows a justfile to be executed
 #   like a script, with `./justfile test`, for example.
 
+set windows-powershell := true
```
Comment (funkyfuture) [inline justfile:5]: i dunno what this does, but it'd heavily irritate me when i see this.

File: justfile [hunk 3]
```diff
@@ -14,6 +16,7 @@ log := "warn"
 
 export JUST_LOG := log
 
+# ltest is installed via `cargo install cargo-limit`
```
Comment (funkyfuture) [inline justfile]: this is implicitly documented in the `install-dev-deps` target as are all *other* development dependencies.

**Review by funkyfuture — COMMENTED** [review]
imho, the proposed changes are introducing inconsistency and don't add any help for others.

File: justfile (hunk 3 above)
Comment (chamons) [reply justfile]: Maybe a comment at the top of the file, or the readme about `install-dev-deps` then? I did not come across it while hacking on this, though I see it now.

File: justfile (hunk 2 above)
Comment (chamons) [reply justfile:5]: See https://github.com/casey/just#windows-powershell

By default, just runs sh on Windows, which means you must build just inside a cygwin or [WSL](https://docs.microsoft.com/en-us/windows/wsl/) prompt. 

This defaults the file to use powershell instead, which works from my limited testing.

File: justfile (hunk 3 above)
Comment (funkyfuture) [reply justfile]: or the `CONTRIBUTING.md`?

File: justfile (hunk 2 above)
Comment (funkyfuture) [reply justfile:5]: i won't be less confused when you tell me things about Windows. ;-)

File: justfile (hunk 3 above)
Comment (chamons) [reply justfile]: That would be _even better_. There are no real descriptive docs there on how to get started.

File: justfile (hunk 2 above)
Comment (casey) [reply justfile:5]: > i dunno what this does, but it'd heavily irritate me when i see this.

This isn't a super helpful comment. There are a lot of Windows users out there!

**Comment by casey** [comment]
I like cargo-limit, but I think it makes the `justfile` more confusing, so I just removed it. I'm not sure about the `set windows-powershell` though, because I suspect that the more complicated recipes don't actually work with `powershell`, and wouldn't want to break them if they're working with `sh` on windows.

**Comment by casey** [comment]
Removed cargo limit in #1199

Sorry for waffling!

**Pull request closed without merging**